### PR TITLE
Add log listener to suite starter

### PR DESCRIPTION
**Pull request by t-persson**
### Applicable Issues

eiffel-community/etos#148
depends-on: eiffel-community/etos-suite-runner#38
depends-on: eiffel-community/etos-library#21

### Description of the Change

Add the ESR log listener to the YAML of ESR.

### Alternate Designs

None

### Benefits

Enables the collection of ETOS logs in the ESR.

### Possible Drawbacks

None

### Sign-off

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.

Signed-off-by: Tobias Persson <tobias.persson@axis.com>

### CODE REVIEW

**Comment by t-persson** [comment]
Failures are expected as we are dependent on an ETOS Library change. Won't merge until 3.1.0 of ETOS Library is released.

**Review by fredjn — DISMISSED** [review]
Perhaps we should consider moving the k8s manifests (esr_yaml.py) into proper .yaml files.

**Comment by t-persson** [comment]
> Perhaps we should consider moving the k8s manifests (esr_yaml.py) into proper .yaml files.

Possibly. Please write an issue. The "problem" with it is that we need to modify the values in a good way as well.
I guess it could be solved by using kustomize or another templating system for ESR.
Won't do anything in this PR at least :)

File: tests/test_suite_starter.py [hunk 1]
```diff
@@ -196,11 +196,11 @@ def test_suite_starter_esr_image(self, mock_job, _):
...
-            f"There is not exactly 1 docker image in kubernetes yaml. There was: {len(image)}",
+            3,
+            f"There is not exactly 3 docker image in kubernetes yaml. There was: {len(image)}",
         )
         self.assertEqual(
-            image[0],
+            image[1],
             self.suite_runner,
             "Docker image sent to ESR is not correct. "
             f"Expected {self.suite_runner!r}, Was {image[0]!r}",
```
Comment (fredjn) [inline tests/test_suite_starter.py:206]:
```suggestion
            f"Expected {self.suite_runner!r}, Was {image[1]!r}",
```

**Review by fredjn — CHANGES_REQUESTED** [review]

**Review by fredjn — APPROVED** [review]

**Pull request merged**